### ISSUE TITLE
Failure to publish qgs file created with local data

### ISSUE THREAD

**Issue by desilvarami** [g3w-suite/g3w-suite-docker#116]
### Checklist

- [X] I've searched through the existing [issues](https://github.com/g3w-suite/g3w-suite-docker/issues) and this bug has never been reported before

### Subject of the issue

Could not publish web services from qgs files created using shp files. I can successfully publish qgs projects created using external PostGIS data in the server.
Please advice, if I am missing some step. In case this is not the correct platform to ask questions, please let me know a link to a such platform.

### Steps to reproduce

- Create qgs file using QGIS Desktop application running in a local computer (ubuntu desktop) using a Shape File (in ~/Downloads/af/AFG_adm0.shp).
- Transfer the same Shape File to G3W remote server (to /shared-volume/media/Downloads/af/AFG_adm0.shp). Included .dbf. prj and .shx files as well.
- Using G3W Admin interface, try to publish the project.
- Results "Layer error on datasource" message as shown in the additional info section.

### Environment

- g3w-suite-docker 3.7.x
- Server OS: Ubuntu 22.04 LTS
- Client Ubuntu 22.04 Desktop with QGIS 3.28 Firenze

### Additional info

Source of the SHP file in Local Desktop which was used to created the qgs project using QGIS Desktop.
![image](https://github.com/g3w-suite/g3w-suite-docker/assets/6550207/eabc4a52-264d-425c-9798-5a6ae9726a64)

Transfer the same dataset to G3W Server as indicated below.
![image](https://github.com/g3w-suite/g3w-suite-docker/assets/6550207/1d57641c-977b-4c67-ae3e-35b89923cd74)

When try to publish the project following error occurs.
![image](https://github.com/g3w-suite/g3w-suite-docker/assets/6550207/db12beec-9c83-45ae-9833-2f7014a7809a)

g3w-suite log file
![image](https://github.com/g3w-suite/g3w-suite-docker/assets/6550207/a1a26cfa-4ca3-494a-88bd-0135ac683c0a)

Please provide some assistance to overcome this issue.

**Comment by wlorenzetti** [comment]
Hi @desilvarami, in accordance with the manual https://g3w-suite.readthedocs.io/en/v3.7.x/datamanagement.html, the geo data must been uploaded in `/shared-volume/project_data`, not in `/shared-volume/media/...`.

To do that inside the G3W-SUITE there is a file manage to upload data on G3W-USITE instance: https://g3w-suite.readthedocs.io/en/v3.7.x/datamanagement.html#geographical-data-synchronization-on-the-server

**Comment by desilvarami** [comment]
Hi @wlorenzetti, thanks a lot for the detailed instructions. I was able to successfully publish geo data following your instructions. The possibility of uploading data via the File Manager was very useful. Previously, I transferred geo data to G3W server by using wget command from my GitHub. I guess that caused the problem (with the file format). Because from the beginning I kept a copy of the dataset in  /shared-volume/project_data folder as well. Really appreciate your guidance.
